content/en/docs/armory-admin/prometheus-monitoring.md

```diff
@@ -177,18 +262,16 @@ Access the Grafana web interface via http://localhost:3000 and use the default g
 
 ## Add Armory dashboards to Grafana
 
-Armory provides some sample dashboards (in JSON format) that you can import into Grafana as a starting point for metrics to graph for monitoring.
-Armory has additional dashboards that are availabe to Armory customers. You can skip this section if you are a Grafana expert.
+Armory provides some sample dashboards (in JSON format) that you can import into Grafana as a starting point for metrics to graph for monitoring. Armory has additional dashboards that are availabe to Armory customers. You can skip this section if you are a Grafana expert.
 
 To import the sample dashboards, perform the following steps:
 
-1. Git clone this repo to your local workstation: (https://github.com/spinnaker/spinnaker-monitoring)
+1. Git clone this repo to your local workstation: (https://gitlab.com/uneeq-oss/spinnaker-mixin)
```

Comment:
Are going to suggest using this dashboards? We now have the plugin version of the old dashboards at:
https://github.com/armory-io/spin-monitoring-dashboards/pull/2

Reply:
This is inline with the previous version of the document. We were referring to the monitoring-daemon OSS dashboards with a statement that Armory provides additional dashboards to our customers. @brian-armory thoughts?

Reply:
@karlomendozaarmory 's link is to the internal Armory GitHub, so that wouldn't be helpful to customers. For now, let's link to the public examples that @christosarvanitis added.